samples/librats-client/client.c

```diff
@@ -0,0 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <getopt.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <unistd.h>
+#include <librats/api.h>
+#include <librats/evidence.h>
+#include <librats/log.h>
+
+#define DEFAULT_PORT	1234
+#define DEFAULT_IP	"127.0.0.1"
+#define DEFAULT_HASH	"1234567812345678123456781234"
+#define MAX_HASH_LENGTH 1024
+
+// startup client端
+int librats_client_startup(char *ip, int port, char hash_value[], bool mutual)
+{
+	const char *hash = hash_value;
+
+	// create socket
+	int sockfd = socket(AF_INET, SOCK_STREAM, 0);
+	if (sockfd < 0) {
+		RATS_ERR("failed to call socket()\n");
+		return -1;
+	}
+
+	struct sockaddr_in s_addr;
+	memset(&s_addr, 0, sizeof(s_addr));
+	s_addr.sin_family = AF_INET;
+	s_addr.sin_port = htons(port);
+
+	// Get the server IPv4 address from the command line call
+	if (inet_pton(AF_INET, ip, &s_addr.sin_addr) != 1) {
+		RATS_ERR("invalid server address\n");
+		return -1;
+	}
+
+	// connect server
+	int connd = connect(sockfd, (struct sockaddr *)&s_addr, sizeof(s_addr));
+	if (connd < 0) {
+		RATS_ERR("failed to call connect()\n");
+		return -1;
+	}
+
+	// send hash to server
+	int len_send = send(sockfd, hash_value, strlen(hash_value), 0);
```

Comment:
I think there is no need to send hash, server and client use the default hash or hash passed by command line